.github/workflows/generate_matrix.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Verible Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+matrix='['
+for item in releasing/*/; do
```

Comment:
This LGTM. 

I was pondering if this should come from the README.md file instead but decided the directory version makes the most sense. Maybe we should have a GitHub Action check which makes sure the README and directories are kept in sync in some way?

Reply:
In #622, the way jobs are defined is to be reworked. Instead of reading directories, a list of images will be provided and subdirs will be removed. It will be easier to handle sync after that.

Reply:
Let's continue in https://github.com/google/verible/pull/622#discussion_r538794804.